models/pathology_nuclei_segmentation_classification/large_files.yml

```diff
@@ -0,0 +1,3 @@
+large_files:
+  - path: "models/model.pt"
+    url: "https://drive.google.com/file/d/1Auwm8YrQCx9EAX_oViNTlJ_JgBb-wbzZ/view?usp=sharing"
```

Comment:
this is wrong file.. it has dict {"net": } all others are using {"model": } as part of state dictionary..